Chummer/data/weapons.xml

```diff
@@ -14843,6 +14843,30 @@
       <hide />
       <spec>Machine Guns</spec>
     </weapon>
+    <weapon>
+      <id>4629315e-9d09-4d04-92ac-3bad46675522</id>
+      <name>Ontario Arms Sling-Shot</name>
+      <category>Underbarrel Weapons</category>
+      <type>Ranged</type>
+      <conceal>6</conceal>
+      <accuracy>6</accuracy>
+      <damage>As Ammo</damage>
```

Comment:
This is a bit of a tricky entry here, especially because it won't get translated. If you absolutely do not want to set a "default" damage code, you should use "Special" here instead, as that will actually get translated.

Reply:
Good point, changed to "Special" instead.